docs/source/conf.py

```diff
@@ -375,7 +375,9 @@ def inject_weight_metadata(app, what, name, obj, options, lines):
             lines.append("")
 
 
-def generate_weights_table(module, table_name, metrics, dataset, include_patterns=None, exclude_patterns=None):
+def generate_weights_table(
+    module, table_name, metrics, dataset, include_patterns=None, exclude_patterns=None, exclude_columns=[]
+):
```

Comment:
I wonder if the introduction of `exclude_columns` is necessary. Could we make the use of `rich_metadata` conditional on the module (aka just eliminate quantized models which currently don't support flops?

Reply:
Let me try this.